core/src/test/java/org/apache/calcite/tools/PlannerTest.java

```diff
@@ -147,6 +148,21 @@ private void checkParseAndConvert(String query,
         + "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa from \"emps\"");
   }
 
+  @Test void testCurrentDatetime() throws Exception {
+    Planner planner = getPlanner(null,
+        SqlParser.config());
+    SqlNode node = planner.parse("select current_datetime");
+    planner.validate(node);
+  }
+
+  @Test void testNiladicForBigQuery() throws Exception {
+    Planner planner = getPlanner(null,
+        SqlParser.config().withConformance(SqlConformanceEnum.BIG_QUERY));
+    SqlNode node = planner.parse("select current_time, current_time(), current_date, "
+            + "current_date(), current_timestamp, current_timestamp(), current_datetime, current_datetime()");
```

Comment:
Maybe we should move the test to `SqlValidatorTest`

Reply:
Ah, sure I can move it.